lib/routes/api/speech-credentials.js

```diff
@@ -51,6 +51,7 @@ const encryptCredential = (obj) => {
     tts_region,
     stt_api_key,
     stt_region,
+    nvidia_speech_endpoint,
```

Comment:
would you mind changing the name of this to `riva_server_uri` ?

Reply:
Yes, I fixed the comment, please help to review again.